### DISCUSSION TITLE
Need to understand ieee14_solar model

### DISCUSSION THREAD

**Discussion by SanjeevKumarGoyal** [CURENT/andes#298] [Q&A] [answered]
Dr. Cui,
it would be really helpful if there is any diagram which gives more information about ieee14_solar model to understand.

Regards,
Sanjeev

**Comment by cuihantao** [comment, marked as answer]
Please see the following documents:

https://www.powerworld.com/files/T12RenewableGeneration_2019.pdf
https://www.wecc.org/Reliability/WECC%20Solar%20Plant%20Dynamic%20Modeling%20Guidelines.pdf

`ieee14_solar` implements `REGC_A`, `REEC_A` and `REPC_A`.
